src/grafana/values/registry1-values.yaml

```diff
@@ -21,4 +21,4 @@ sidecar:
   image:
     registry: registry1.dso.mil
     repository: ironbank/kiwigrid/k8s-sidecar
-    tag: 1.30.10
+    tag: 2.1.4
```

Comment:
We'll want to avoid the 2.x stream for now pending upstream fixes. Probably need to get a renovate config update to block these.